packages/cli/demo/cat.js

```diff
@@ -175,20 +186,47 @@ const template = `
 <div id="controls">
   <button id="cat">🐈‍⬛</button>
   <button id="chat-button">Chat</button>
+  <button id="eval-button">Eval</button>
 </div>
 
 <div id="chat-frame" class="frame">
   <div id="chat-window" class="window">
     <label for="chat-to">To:&nbsp;<select id="chat-to"></select></label>
     <p><input type="text" id="chat-message">
-    <p id="endowments">
     <p id="chat-error" class="error">
     <p><button id="chat-discard-button">Discard</button>
     <button id="chat-send-button">Send</button>
   </div>
 </div>
+
+<div id="eval-frame" class="frame">
```

Comment:
This element is insufficiently reactive; when I open the console in Chrome on my system, the eval window adjusts left and its form elements become invisible.

Reply:
☢️ irradiated